exchanges/lbank/lbank_wrapper.go

```diff
@@ -95,6 +96,21 @@ func (l *Lbank) SetDefaults() {
 		},
 		Enabled: exchange.FeaturesEnabled{
 			AutoPairUpdates: true,
+			Kline: kline.ExchangeCapabilitiesEnabled{
+				Intervals: map[string]bool{
+					kline.OneMin.Word():     true,
+					kline.FiveMin.Word():    true,
+					kline.FifteenMin.Word(): true,
+					kline.ThirtyMin.Word():  true,
+					kline.OneHour.Word():    true,
+					kline.FourHour.Word():   true,
+					"eighthour":             true,
```

Comment:
Perhaps add an official kline type for this, similar to what you've done for Kraken's 15d

Reply:
Yep will do that originally i wanted to keep one use exchange ones on the exchange but changed that later and forgot to update lbank